### PR TITLE
Adding signature safety margin on DKG result

### PR DESCRIPTION
**Pull request by dimpar**
Close https://github.com/keep-network/keep-core/issues/1585

In this PR we are increasing the DKG nofail signature threshold to 75%.  It gives us a 25% safety margin for relay entry signing.

### CODE REVIEW

**Comment by pdyraga** [comment]
@eth-r @ngrinkevich would love to have your 👀 on this one.

File: solidity/test/random_beacon_operator/TestPublishDkgResult.js [hunk 1]
```diff
@@ -5,28 +5,29 @@ const generateTickets = require('../helpers/generateTickets');
...
-const {createSnapshot, restoreSnapshot} = require("../helpers/snapshot.js")
-const {contract, accounts, web3} = require("@openzeppelin/test-environment")
-const {expectRevert, time} = require("@openzeppelin/test-helpers")
+const { createSnapshot, restoreSnapshot } = require("../helpers/snapshot.js")
+const { contract, accounts, web3 } = require("@openzeppelin/test-environment")
+const { expectRevert, time } = require("@openzeppelin/test-helpers")
 const stakeDelegate = require('../helpers/stakeDelegate')
 
-describe('KeepRandomBeaconOperator/PublishDkgResult', function() {
+describe.only('KeepRandomBeaconOperator/PublishDkgResult', function () {
```
Comment (dimpar) [inline solidity/test/random_beacon_operator/TestPublishDkgResult.js:13]: only

File: pkg/beacon/relay/dkg/result/submission.go [hunk 2]
```diff
@@ -61,11 +61,15 @@ func (sm *SubmittingMember) SubmitDKGResult(
 		)
 	}
 
-	if len(signatures) < config.HonestThreshold {
+	// Chain rejects the result if it has less than 25% safety margin.
+	// If there are not enough signatures to preserve the margin, it does not
```
Comment (dimpar) [inline pkg/beacon/relay/dkg/result/submission.go:65]: Maybe we can also mention in the comment that we'd need total of 75% signatures?

Comment (pdyraga) [reply pkg/beacon/relay/dkg/result/submission.go:65]: Technically, from the client's perspective, it doesn't have to be 75% if the on-chain threshold parameter is different from what we currently have. That's also why do we need to re-calculate this parameter here.

**Comment by eth-r** [comment]
The solidity side looks good to me

**Comment by pdyraga** [comment]
And everything else looks good to me too. Let's merge.

**Pull request merged**